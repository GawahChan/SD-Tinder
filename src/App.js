import React from "react";
import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
import { HOME_URL, DECK_URL } from "./Constants";

import { GlobalStyle } from "./App.styles";
import Menu from "./Components/Menu/Menu";
import Deck from "./Components/Deck/Deck";
import Results from "./Components/Results/Results";

function App() {
  return (
    <Router>
      <div>
        <GlobalStyle />
        <Menu />
        <Switch>
          <Route exact path={HOME_URL}>
            <Results />
          </Route>
          <Route exact path={DECK_URL}>
            <Deck />
          </Route>
        </Switch>
      </div>
    </Router>
  );
}

export default App;
