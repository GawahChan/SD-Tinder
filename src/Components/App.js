import React from 'react';
import Card from './Card/Card.js';
import Data from '../Data.js';
import './App.css';

function App() {
  return (
    <div className="app">
      <h1>Stunt Double App</h1>
        {
          Data.map(stuntDouble => {
            return (
              <Card 
                key={stuntDouble.id}
                name={stuntDouble.name}
                age={stuntDouble.age}
                location={stuntDouble.location}
                bio={stuntDouble.bio}
              />
            );
          })
        }
    </div>
  );
}

export default App;
