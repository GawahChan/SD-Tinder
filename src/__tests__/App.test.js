import React from 'react';
import { shallow } from 'enzyme';
import App from '../Components/App';

describe('App', () => {
    const app = shallow(<App />);

    it('renders correctly', () => {
        expect(app).toMatchSnapshot();
    });
})