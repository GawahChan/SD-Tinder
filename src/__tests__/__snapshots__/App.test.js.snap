// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  className="App"
>
  <h1>
    Stunt double App
  </h1>
  <Card
    age="30"
    bio="aspiring milf"
    key="0"
    location="LA"
    name="Mila Kunis"
  />
</div>
`;
